modules/cms/routes.php

```diff
@@ -22,7 +22,7 @@
      * The CMS module intercepts all URLs that were not
      * handled by the back-end modules.
      */
-    Route::any('{slug}', 'Cms\Classes\CmsController@run')->where('slug', '(.*)?')->middleware('web');
+    Route::any('{slug?}', 'Cms\Classes\CmsController@run')->where('slug', '(.*)?')->middleware('web');
```

Comment:
Why was this required?

Reply:
The home page wouldn't load without it, as the home page has no slug. :)

Reply:
But home pages work fine right now, so what changed?

Reply:
This is what happens without the change:
![image](https://user-images.githubusercontent.com/15900351/72707513-8248cc00-3b9b-11ea-98c1-63b9f9f21fcf.png)
I can only surmise that Laravel has fixed something with regards to the difference between required URL parameters and optional ones.

Reply:
Hmm, odd. Oh well, should be fine.